setup.py

```diff
@@ -122,6 +122,17 @@
     author_email='rbean@redhat.com',
     url='https://github.com/fedora-infra/fedmsg/',
     license='LGPLv2+',
+    classifiers=[  # https://pypi.python.org/pypi?%3Aaction=list_classifiers
+        'Development Status :: 3 - Alpha',
```

Comment:
The version numbering wiki thing sounds pretty similar to semver to me. It's not exact and doesn't go into as much detail, but the basic idea sounds the same to me. FWIW, I've been trying to just follow semver for Bodhi because I didn't know about that document, though I also don't think I've done anything that that would "disagree" with it either.

Another idea to confuse things even more: what if we just go ahead and mark fedmsg as being a 1.0 and call it stable? Then, we can always make a 2.0 when we are ready to change the API. I don't think that would be practically different than keeping it as a 0.y for a long time and then changing it for a 1.0 or a 2.0. Tons of apps are currently relying on its API, so going ahead and calling it a 1.0 could be a nice statement for all the other apps. And we can still make 2.0 when we need to change it. Just an idea.